src/js/index.js

```diff
@@ -22,6 +22,9 @@ const getCurrentPage = () => {
   const clean = window.location.href.split('#');
   const path = clean[0].split('/');
   const location = path.slice(-1)[0] || 'home';
+  if (!pageData[location] || location === 'common') {
```

Comment:
Remove the .html extension before checking this condition, so that when you put in the URL for example about.html it renders the right elements without redirecting to the / path

Reply:
What I mean is that the `location.replace('.html', '');` should be done before this conditional. Like so,
```suggestion
  const location = path.slice(-1)[0].replace('.html', '') || 'home';
  if (!pageData[location] || location === 'common') {
```
And the return of the function would be only the location variable, without the replace.